server/src/lsp-server.ts

```diff
@@ -758,7 +758,9 @@ export class LspServer {
         }
         const result: lsp.DocumentHighlight[] = [];
```

Comment:
probably better move the normalization of the `file` up to here, like

```js
const normalizedFilePath = path.normalize(file);
```

Reply:
Done, thanks.